models/attachments.js

```diff
@@ -129,6 +129,9 @@ if (Meteor.isServer) {
 
       const fileObj = Attachments.findOne({_id: fileObjId});
       moveToStorage(fileObj, storageDestination, fileStoreStrategyFactory);
+
+      // since Meteor-Files 2.1.0 the filename is truncated to 28 characters, so rename the file after upload to the right filename back
```

Comment:
@mfilser @xet7 I'm open to have a pass-through option for sanitization. Believe me there are not pleasant and real cases behind this sanitization

Reply:
like an option to overwrite and pass your own `sanitize` method into the *FilesCollection* constructor

Reply:
@dr-dimitru 

Yes, it would be very nice. Also new feature for checking uploaded attachments for viruses etc would expect filenames to be sanitized:

https://github.com/wekan/wekan/pull/4637

Reply:
There is also WeKan setting to validate attachment by file size and type:

https://github.com/wekan/wekan/pull/4618

Reply:
@mfilser 

What do you think, would that `sanitize` method of `FilesCollection` constructor be better?

Reply:
@xet7 @dr-dimitru 
That would be great to have additional options at `config` variable of [FilesCollection constructor](https://github.com/veliovgroup/Meteor-Files/blob/master/docs/constructor.md)

@dr-dimitru 
Thank you very much for your great framework, without it, it would be much more diffucult or impossible to make all this features in Wekan !